omod/src/main/webapp/resources/htmlforms/071-IntegratedAntenatalRegister.xml

```diff
@@ -682,8 +931,8 @@
                         <p>
                             <label>IPT Dose</label>
                             <obs conceptId="1da3cb98-59d8-4bfd-b0bb-c9c1bcd058c6"
-                                 answerConceptIds="0192ca59-b647-4f88-b07e-8fda991ba6d6,f1d5afce-8dfe-4d2d-b24b-051815d61848,f29f43c6-076a-4f70-9ae8-4563ac3fda80,dca06bae-30ab-102d-86b0-7a5022ba4115,fca28768-50dc-4d6b-a3d2-2aae3b376b27,b0439b8e-469e-43e6-9dda-9b6b49f2147b"
-                                 answerLabels="1,2,ND,C,CTX,CTX✔"
+                                 answerConceptIds="0192ca59-b647-4f88-b07e-8fda991ba6d6,f1d5afce-8dfe-4d2d-b24b-051815d61848,a5497b5a-7da1-42d2-9985-b5ec695b4199,f29f43c6-076a-4f70-9ae8-4563ac3fda80,dca06bae-30ab-102d-86b0-7a5022ba4115,fca28768-50dc-4d6b-a3d2-2aae3b376b27,b0439b8e-469e-43e6-9dda-9b6b49f2147b"
+                                 answerLabels="1 - IPT1,2 - IPT2,3 - IPT3,ND - Not Due,C - Complete,CTX - Initial  septrin prescrition,CTX✔ - Septrin prescribed on prev visit"
```

Comment:
Change the tick to &#10003; as its breaking the forms when saving to different databases